src/libstore/ssh-store.cc

```diff
@@ -12,23 +10,22 @@
 
 namespace nix {
 
-struct SSHStoreConfig : virtual RemoteStoreConfig, virtual CommonSSHStoreConfig
+SSHStoreConfig::SSHStoreConfig(
+    std::string_view scheme,
+    std::string_view host,
+    const Params & params)
+    : StoreConfig(params)
```

Comment:
should this validate it's ssh-ng ?

Reply:
Oh can't do that because its `mounted-ssh-ng` for subclass (which is bad, but we cannot yet fix)